.github/workflows/docs.yml

```
@@ -0,0 +1,35 @@
+name: Documentation
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
```

You must explicitly elevate the permissions of this job to push to the repo, as the GITHUB_TOKEN provided is otherwise not granting you access to do that by default in our jupyterhub org.

---

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idpermissions

I think what you need is...

```suggestion
    permissions:
      contents: write
    runs-on: ubuntu-20.04
```